### ISSUE TITLE
Implement the `-p` argument

### ISSUE THREAD

**Issue by FreeMasen** [dtolnay/cargo-expand#59]
Currently `cargo expand -p some-workspace-crate` fails with the following 

```
Found argument '-p' which wasn't expected, or isn't valid in this context
```

It would be nice to expand macros in other workspace crates.

**Comment by dtolnay** [comment]
Thanks, fixed in 0.4.11.

**Comment by FreeMasen** [comment]
Ohh my goodness, thank you!
